Mayhemfile

```diff
@@ -0,0 +1,16 @@
+# Namespaced project name that the target belongs to
```

Comment:
Multiple commands are not supported; please use only one or separate into multiple Mayhemfiles, use a generic project name, and omit the image.